docs/usermanual.md

```diff
@@ -98,17 +98,130 @@ hello
+tools available online like [this one](https://string-functions.com/string-hex.aspx)
+
+##### NGSI-v2 and NGSI-LD Measure reporting
+
+It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with payloadType
+`ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure attributes and
+`id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used, as well as
+`actionType`.
+
+Examples of these `ngsi-v2` payloads are:
+
+```
+ {
+     "actionType": "append",
+     "entities": [
+        {
+          "id": "MyEntityId1",
+          "type": "MyEntityType1",
+          "attr1": { type: "Text", "value": "MyAttr1Value"},
+          ...
+        },
+        ...
+    ]
+ }
+```
+
+and (NGSI-v2 plain entities array format):
+
+```
+ [
+    {
+          "id": "MyEntityId1",
+          "type": "MyEntityType1",
+          "attr1": { "type": "Text", "value": "MyAttr1Value"},
+          ...
+    },
+    ...
+ ]
+```
+
+Examples of these `ngsi-ld` payloads are:
+
+```
+ {
+    actionType: 'APPEND',
+    entities: [
+                    {
+                        id: 'urn:ngsi-ld:ParkingSpot:santander:daoiz_velarde_1_5:3',
+                        type: 'ParkingSpot',
+                        status: {
+                            type: 'Property',
+                            value: 'free',
+                            observedAt: '2018-09-21T12:00:00Z'
+                        },
+                        category: {
+                            type: 'Property',
+                            value: ['onstreet']
+                        },
+                        refParkingSite: {
+                            type: 'Relationship',
+                            object: 'urn:ngsi-ld:ParkingSite:santander:daoiz_velarde_1_5'
+                        },
+                        name: {
+                            type: 'Property',
+                            value: 'A-13'
+                        },
+                        location: {
+                            type: 'GeoProperty',
+                            value: {
+                                type: 'Point',
+                                coordinates: [-3.80356167695194, 43.46296641666926]
+                            }
+                        },
+                        '@context': [
+                            'https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld',
+                            'https://schema.lab.fiware.org/ld/context'
+                        ]
+                    }
+    ]
+ }
+```
 
-Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte received in 
-the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple tools available 
-online like [this one](https://string-functions.com/string-hex.aspx)
+and
+
+```
+ {
```

Comment:
I understand this is a single entity. The array case (supported in NGSI-v2) is not supported in NGSI-LD case?

Reply:
In NGSI-LD case both cases are also supported

Reply:
Should we include here the array case (to align with the NGSIv2 examples)?

Reply:
fixed in https://github.com/telefonicaid/iotagent-json/pull/779/commits/a28801ab572b9422b5d1db716b4a3204a7022f4d